application-flashmessages-ui/src/main/resources/Flash/DisplayFlashMessages.xml

```diff
@@ -154,8 +154,15 @@
   #set($msgDoc = $xwiki.getDocument($msg.doc))
   #set($msgValObj = $msgDoc.getObject('Flash.FlashValidationClass'))
   #set($msgValList = $msgValObj.getProperty('flash').getValue())
+  #set($msgDateSeen = $msgValObj.getProperty('seenDate').getValue())
+  #set($dateToday = $datetool.format('yyyy-M-d',$dateNow))
```

Comment:
`yyyy-M-d` doesn't look like the default date format (`dd/MM/yyyy HH:mm:ss` is used when creating an XClass with a Date property). Not sure why did you choose the other one.

Reply:
I just needed to get the day to see if it's different. So `dd/MM/yyyy` would be better? Or I should also consider the time?

Reply:
Sure, take only what you need.

Reply:
Ok, thanks